flocker/node/test/test_script.py

```diff
@@ -416,7 +416,7 @@ def test_main_starts_service(self):
         """
         service = Service()
         options = ZFSAgentOptions()
-        options.parseOptions([b"example.com"])
+        options.parseOptions([b"1.2.3.4", b"example.com"])
```

Comment:
OTOH it's less clear that they're required? We can change that in some other issue if you want. In practice I doubt humans will run these much.